examples/purchaseTesterTypescript/.ruby-version

```diff
@@ -1 +1 @@
-3.0.6
+3.1.2
```

Comment:
I did this change because our current version of activesupport (dependency of cocoapods) didn't work with <3.1.0.